raiden/transfer/state.py

```diff
@@ -4,8 +4,9 @@
 from raiden.encoding.format import buffer_for
 from raiden.encoding import messages
 from raiden.transfer.architecture import State
+from raiden.utils import pex, typing
```

Comment:
@LefterisJP Seems like you chose the wrong diff, `lpex` and `sha3` are used in the file.

Reply:
I know -- I am on it. No worries. Conflict resolution does not always work out fine :)